modules/configuring-default-route-timeouts-globally.adoc

```diff
@@ -0,0 +1,100 @@
+
+.Procedure
+
+You can configure the route timeouts globally by updating the `ROUTE_HAPROXY_TIMEOUT` environment value in your `serverless-operator` subscription and updating the `max-revision-timeout-seconds` in your `KnativeServing` custom resource (CR). This applies the timeout changes across all Knative services, and you can deploy services with specific timeouts up to the maximum value set.
+
+. In your subscription, set the value of `ROUTE_HAPROXY_TIMEOUT` to your required timeout in seconds:
++
+.Example of setting the `ROUTE_HAPROXY_TIMEOUT` value to 900 seconds
+[source,terminal]
+----
+$ oc patch subscription.operators.coreos.com serverless-operator -n openshift-serverless --type='merge' -p '{"spec": {"config": {"env": [{"name": "ROUTE_HAPROXY_TIMEOUT", "value": "900"}]}}}'
+----
++
+`ROUTE_HAPROXY_TIMEOUT` is an enviroment managed by the Serverless operator and it is by default set to `600`. This subscription update will cause pods to be redeployed in the `openshift-serverless` namespace.
+
+[NOTE]
+====
+If you created your routes manually and disabled auto-generation with the annotation `serving.knative.openshift.io/disableRoute`, you can configure the timeouts directly in the route definitions.
+====
+
+. Set the maximum revision timeout as follows in the `KnativeServing` custom resource (CR):
++
+.Example of `max-revision-timeout-seconds` set to 900 seconds
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1beta1
+kind: KnativeServing
+metadata:
+  name: knative-serving
+spec:
+  config:
+    defaults:
+      max-revision-timeout-seconds: "900"
+----
++
+Serverless operator automatically adjusts the activator's `terminationGracePeriod` to this value to avoid request termination in cases where activator pods are being terminated themselves.
++
+You can verify that the timeout has been set by running the following command: 
++
+[source,terminal]
+----
+$ oc get deployment activator -n knative-serving -o jsonpath="{.spec.template.spec.terminationGracePeriodSeconds}"
+900
+----
+
+. Now you can deploy a Knative service with the desired timeouts less or equal to the maximum set:
++
+.A service definition with timeouts set to 800 seconds
+[source,yaml]
+----
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name:
+spec:
+  template:
+    spec:
+      timeoutSeconds: 800
+      responseStartTimeoutSeconds: 800
+----
+
+. Depending on your cloud provider, you may also need to set or adjust the load balancer timeout:
++
+.Example of load balancer timeout adjustment for AWS Classic LB 
+[source,terminal]
+----
+$ oc -n openshift-ingress-operator patch ingresscontroller/default --type=merge --patch='{"spec":{"endpointPublishingStrategy": {"type":"LoadBalancerService", "loadBalancer": {"scope":"External", "providerParameters":{"type":"AWS", "aws": {"type":"Classic", "classicLoadBalancer": {"connectionIdleTimeout":"20m"}}}}}}}'
+ingresscontroller.operator.openshift.io/default patched
+----
+
+[IMPORTANT]
```

Comment:
I would move this up to the start of the procedure and provide more details. Does this adjustment need to be done at the start of the procedure or at this moment in the procedure? If you leave it here, you need a `+` on line 78 to indent it.